### ISSUE TITLE
House Preview vs Model Preview (Models Type of String)

### ISSUE THREAD

**Issue by buzzdude** [smeighan/xLights#177]
Hi Guys

House Preview shows some RGB colours (wrong) vs Model Preview (correct), based on the "Type of String" set in Model tab of Model List.  In the picture below RED models are set to RGB and Green models are set to GBR, all should show RED.  Colours also show incorrectly in the Preview TAB.

Thanks

Shane
![untitled](https://cloud.githubusercontent.com/assets/11732991/7018594/d9e1307a-dd49-11e4-9dfe-d3b01911b9a0.png)

**Comment by AzGilrock** [comment]
I'm not sure how to reproduce what you have going on here.  I created a single string that was RGB and another that was GBR and put a red effect on both and got matching results in the Model Preview and the House Preview.  Since the model preview can only show one model you must have combined them as a whole house model?

**Comment by buzzdude** [comment]
Apologies, yes, it's the whole house model that is showing incorrect.  You can reproduce the problem if you use the GBR and create a whole house.  The two previews show differently.
![untitled](https://cloud.githubusercontent.com/assets/11732991/7061420/faebe0bc-ded4-11e4-84d5-91564342058a.png)

**Comment by dkulp** [comment]
Yes.  This is a bug in the whole house model.   The whole house model currently only works "well" with RGB nodes.   It has issues with non-RGB nodes as well as single color strands.  My suggestion is to just use RGB nodes and set the string ordering at the controller (where it belongs :) ).

**Comment by AzGilrock** [comment]
I was able to reproduce it but I also discovered ways the user can create a whole house model that is technically invalid.  And that's by using models that have channel numbers in common.  In my case I created an RGB string and a GBR string but I had both strings starting at channel 1 so whoever renders last decides the rgb color ordering for that channel.  Also if you have two strings mapped to common channels and group them in a WHM then when you drop a multicolored effect on them the strings will fight each other on who get to define the color for that channel.

So even though our WHM may not be capturing the node's rgb order there are still ways the user can mess things up even after we fix that.
